catalyst_voices/lib/pages/account/account_page.dart

```diff
@@ -35,7 +38,25 @@ final class AccountPage extends StatelessWidget {
                       AccountRole.drep,
                     ],
                     defaultRole: AccountRole.voter,
-                    onRemoveKeychain: () => debugPrint('Keychain removed'),
+                    onRemoveKeychain: () async {
+                      await VoicesDialog.show<void>(
+                        context: context,
+                        builder: (context) {
+                          return DeleteKeychainDialog(
+                            onRemoveKeychainConfirmed: () async {
+                              // TODO(Jakub): remove keychain
+                              Navigator.of(context).pop();
+                              await VoicesDialog.show<void>(
```

Comment:
Problem here is that you're using `context` from `DeleteKeychainDialog` dialog, which is going to be disposed soon. Probably its not end of the world but this is not good practice and nesting is bad :D